templates/csharp/ai-assistant-bot/Program.cs.tpl

```diff
@@ -51,12 +51,23 @@ builder.Services.AddTransient<IBot>(sp =>
         .WithStorage(sp.GetService<IStorage>())
         .Build();
 
+    app.OnConversationUpdate("membersAdded", async (turnContext, turnState, cancellationToken) =>
+    {
+        var welcomeText = "How can I help you today?";
+        foreach (var member in turnContext.Activity.MembersAdded)
+        {
+            if (member.Id != turnContext.Activity.Recipient.Id)
+            {
+                await turnContext.SendActivityAsync(MessageFactory.Text(welcomeText), cancellationToken);
+            }
+        }
+    });
 
     // Register AI actions
     app.AI.ImportActions(new ActionHandlers());
 
     // Listen for user to say "/reset".
```

Comment:
comment or another other doc/readme fix?

Reply:
Good catch, will fix it